src/Dev/SapphireTest.php

```diff
@@ -280,6 +285,50 @@ protected function setUp()
         Email::config()->remove('bcc_all_emails_to');
     }
 
+
+
+    /**
+     * Helper method to determine if the current test should enable a test database
+     *
+     * @param $fixtureFiles
+     * @return bool
+     */
+    protected function setupDatabaseForCurrentTest($fixtureFiles): bool
```

Comment:
PHP 5.6 compatible syntax please. :)

Reply:
`setup` replies a verb rather than an answer to a question. Perhaps `isDatabaseRequired()`?